src/lib/components/input/input.svelte

```diff
@@ -102,7 +102,7 @@
 		</div>
 		{#if error}
 			<div class="error-message">
-				{@render error()}
+				{error}
```

Comment:
This will just print strings unformatted, like `a,b,c`.
```suggestion
				{error.join(', ')}
```